File: resources/js/3store.ts
import { reactive } from 'vue';
import * as _types from './0types.ts';
import * as _constants from './1constants.ts';


export const store = reactive<_types.tstore>({
    'tasks': [],
    'config': {
        'dataFilePath': _constants.defaultDataFilePath,
        'shouldNotify': true,
        'shouldMinimizeToTrayOnQuit': true,
    }
});

File: fyne/src/3store.go
package src

import "github.com/google/uuid"

var Store = TStore{
	TaskGroups: []TTaskGroup{
		{
			Id:      uuid.NewString(),
			Version: TaskFileVersion,
			Tasks:   []TTask{},
		},
	},
	Config: TConfig{
		DataFilePath:               DefaultDataFileName,
		ShouldNotify:               true,
		ShouldMinimizeToTrayOnQuit: true,
	},
}
